run/john.conf

```diff
@@ -376,6 +376,14 @@ Frequency = 141
 # This turns off all 10 units (resulting in a timeout).
 #Config1_04A36E0FD6_0 = \xff\x03
 
+[ZTEX:sha256crypt]
+# Design tools reported possible frequency is 166.3 but tested boards
+# miss guesses, often fail unless frequency is decreased.
+#Frequency = 165
+Frequency = 135
+#TargetRounds = 1000000
+
+
```

Comment:
Extra linefeed here.

Reply:
Thank you!
The documentation definitely requires an update / correction. I'm preparing a PR to include topics you've commented on. Maybe some other items require more detailed description, such as resource usage, performance of 16-bit CPU in sha256crypt-ztex vs 32-bit CPU in sha512crypt-ztex.
Regarding extra linefeed in run/john.conf:L386, I was unsure on style in john.conf. Sometimes sections that refer to different subsystems are separated with 2 empty lines, i.e. L419, L474.

Reply:
You're right about the extra linefeed, I missed the fact this was the very last ZTEX entry, and is followed by OpenCL stuff. Makes sense.